src/wallet/wallet.cpp

```diff
@@ -222,6 +219,17 @@ bool CWallet::LoadCryptedKey(const CPubKey &vchPubKey, const std::vector<unsigne
     return CCryptoKeyStore::AddCryptedKey(vchPubKey, vchCryptedSecret);
 }
 
+void CWallet::UpdateTimeFirstKey(int64_t nCreateTime)
+{
+    AssertLockHeld(cs_wallet);
+    // Cannot determine birthday information, so set the wallet birthday to the
+    // beginning of time.
+    if (nCreateTime <= 1)
```

Comment:
Maybe I'm misreading the code, but the comment `Cannot determine birthday information` seems to refer to the if case below it being true; I would prefer comments being inside the block in this case.  I would probably also include brackets after doing so as the comment makes the single-line conditional state less obvious.